app/services/badges/award.rb

```diff
@@ -1,6 +1,6 @@
 module Badges
   class Award
-    def self.call(user_relation, slug, message_markdown)
+    def self.call(user_relation, slug, message_markdown, include_default_description = true) # rubocop:disable Style/OptionalBooleanParameter
```

Comment:
nit: kwargs is preferred because name args causes less confusion.
```suggestion
    def self.call(user_relation, slug, message_markdown, include_default_description: true)
```

Reply:
Yeah @maestromac good call. I went this route as slightly expedited to see if we could get this merged yesterday as it meant fewer changes elsewhere, but I made the adjustment to be compatible with the guideline now.